docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -34,7 +34,7 @@ management policies to determine some placement preferences on the node.
 ### Configuration
 
 The CPU Manager is introduced as an alpha feature in Kubernetes v1.8. It
-must be explicitly enabled in the kubelet feature gates:
+must be explicitly enabled in the kubelet feature gates before v1.10:
```

Comment:
If this is graduating to beta, should this be reworded? Perhaps something like:

> The CPU Manager is an alpha feature in Kubernetes 1.8. It was enabled by default as a beta feature in Kubernetes 1.10

If the `--feature-gates`is no longer needed to enable it, then remove the rest of the content about enabling it that way?

Reply:
fixed.